docs/design/queue-guarantee-resource-reservation-design.md

```diff
@@ -0,0 +1,159 @@
+# Volcano Resource Reservation For Queue
+
+@[qiankunli](https://github.com/qiankunli); Oct 11rd, 2021
+
+## Motivation
+
+In my case, we use volcano to schedule training job(tfjob/pytorchjob/vcjob) in k8s cluster, there are many groups such as ad/recommend/tts, a queue represents a group.
+In order to ensure the full utilization of resources, we generally do not configure `queue.capability`. but this will cause a queue to running out all resources, and make the new job of other queue unable to execute.
+so we want to reserve some resources for a queue, so that any new job in the queue can be submitted immediately.
```

Comment:
If I only have one queue. Will this feature play any role?

Reply:
the queue can running out all the resources in k8s cluster